### PR TITLE
Day 5 basic

### LINKED ISSUE #4: Reading images day 5 %%time
The notebook reading images day 5 wfm except for the cell

```
# Do not run this cell in Google Colab
%%time
image_location = 'images/mri.ome.zarr/s0'
data = load_binary_from_local_with_data(image_location)
print(data.shape)
```
Response

``UsageError: Line magic function `%%time` not found.``

Can we remove or replace ?

### PR DESCRIPTION
**Pull request by jburel**
This PR
* Creates a basic env used by readimage notebook and for supporting dask notebook
* Fixes https://github.com/ome/EMBL-EBI-imaging-course-04-2024/issues/4

### CODE REVIEW

File: Day_5/Reading_images.ipynb [hunk 1]
```diff
@@ -189,36 +180,46 @@
...
-   "execution_count": null,
-   "id": "5a83e9f7",
+   "cell_type": "markdown",
+   "id": "8964ec56-beb9-48ee-af2e-14d3ae9a0bbd",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "# Do not run this cell in Google Colab\n",
-    "interact(update, z= widgets.IntSlider(value=0, min=0, max=data.shape[n]-1, step=1, description=\"Select slice\", continuous_update=False))"
+    "**Do not run the cell below is running in Colab**"
```
Comment (pwalczysko) [inline Day_5/Reading_images.ipynb]:
```suggestion
    "**Do not run the cell below if in Colab**"
```

File: Day_5/Reading_images.ipynb [hunk 2]
```diff
@@ -133,39 +142,21 @@
     "**Exercise**: if you have generated a file locally as part of the [conversion workflow](Conversion.ipynb), change the ``image_location`` parameter to, for example, ``/tmp/conversion_out/B4_C3.ome.zarr/0/0``."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "0a73aac9-5187-4f46-97e8-66fff3a645d3",
+   "metadata": {},
+   "source": [
+    "**Do not run the cell below is running in Colab**"
```
Comment (pwalczysko) [inline Day_5/Reading_images.ipynb]:
```suggestion
    "**Do not run the cell below if in Colab**"
```

**Comment by pwalczysko** [comment]
two suggetions, the Dask and Reading images ntbks day 5 work as expected on M1 and Colab.

One question: Even though jupyter is installed already via the mamba env command, I still have to set the kernel, is that right ?

RFE: Could we name the ``environment.yml`` --> ``environment_cellpose.yml`` ? Or is this not accurrate ?

**Comment by pwalczysko** [comment]
Otherwise lgmt.

**Comment by jburel** [comment]
This is done in another PR. so it is easier to manage

**Pull request merged**